src/main/java/ua/softserveinc/tc/constants/ValidationConstants.java

```diff
@@ -67,6 +67,7 @@
     public static final String EVENT_DESCRIPTION_LENGTH_ERROR_MSG = "Description  should be less than 250 characters";
     public static final String EVENT_INACTIVE_ROOM_ERROR_MSG = "You can't create event on inactive room. Please contact to admin";
     public static final String EVENT_END_MUST_BIGGER_ONE_MINUTE_MSG = "End time must be bigger than start at least one minute";
+    public static final String EVENT_RECCURRENT_END_MUST_BIGER_ONE_DAY_MSG = "End date must be bigger than end at least one day";
```

Comment:
end time must be bigger than end.. ? 
dont really understand meaning of this sent.